src/app.module.ts

```diff
@@ -2,10 +2,31 @@ import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
 import { HelloWorldModule } from './hello-world/hello-world.module';
+import { AuthModule } from './auth/auth.module';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { UserEntity } from './user/user.entity';
+import { AuthController } from './auth/auth.controller';
+const dotenv = require('dotenv');
+dotenv.config();
 
 @Module({
-  imports: [HelloWorldModule],
-  controllers: [AppController],
+  imports: [
+    TypeOrmModule.forRoot({
+      type: 'mysql',
+      host: 'localhost',
+      port: 3306,
+      username: process.env.DATABASE_USERNAME,
+      password: process.env.MYSQL_PASSWORD,
+      database: process.env.DATABASE_LINK,
```

Comment:
(suggestion; non-blocking): We should consider using consistent environment variable name conventions. We have both `DATABASE_` and `MYSQL_`. Typically, it would be one or the other.